crates/rethnet_eth/src/utils.rs

```diff
@@ -38,3 +39,39 @@ where
 
     keccak256(&eth_message)
 }
+
+/// construct a SecretKey from a hex-encoded string
+pub fn private_key_from_string(s: &str) -> SecretKey {
```

Comment:
This already exists in `signature.rs`: https://github.com/NomicFoundation/hardhat/blob/rethnet/main/crates/rethnet_eth/src/signature.rs

Reply:
I changed to using the already-existing one, and I also ported my test over to that module as well.